site/src/docs/components/cookie-consent/code.mdx

```diff
@@ -0,0 +1,119 @@
+---
+slug: '/components/cookie-consent/code'
+title: 'CookieConsent - Code'
+---
+
+import { Link } from 'hds-react';
+import Playground from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Code
+
+### Code examples 
+
+### Modal
+
+```jsx
+import { CookieModal } from 'hds-react';
+
+export const SimpleModalVersion = () => {
+  const [language, setLanguage] = React.useState('en');
+  const onLanguageChange = (newLang) => setLanguage(newLang);
+  const contentSource = {
+    siteName: 'Test site',
+    currentLanguage: language,
+    optionalCookies: {
+      cookies: [
+          {
+            commonGroup: 'statistics',
+            commonCookie: 'matomo',
+          },
+        ],
+    },
+    language: {
+      onLanguageChange,
+    },
+    focusTargetSelector: '#focused-element-after-cookie-consent-closed',
+    onAllConsentsGiven: (consents) => {
+      // called when consents are saved
+      // handle changes like:
+      if (!consents.matomo) {
+        // stop matomo tracking
+      }
+    }
+  };
+
+  const Application = () => {
+    return (
+      <div>
+        <h1 id="focused-element-after-cookie-consent-closed" tabIndex={0}>
+          Simple cookie consent example
+        </h1>
+      </div>
+    );
+  };
+
+  return (
+    <>
+      <CookieModal contentSource={contentSource} />
+      <Application />
+    </>
+  );
+};
+```
+
+<Link href="/storybook/react/?path=/story/components-cookieconsent--modal-version">View working modal demo in Storybook</Link>
```

Comment:
The link is broken in demo environment. Add  gatsby`withPrefix` function to href, and `openInNewTab` prop for the Link component. As the storybook goes out of the docsite app, I think we should open it into a new tab so user can more easily get back to docsite.

Reply:
Fixed